README.md

```diff
@@ -10,15 +10,36 @@ Jazz Hands is a automatic build and dev-run tool for Django projects with modern
 * Runs your dev server and automatically rebuilds front-end assets when change are detected!
 * Is Magick :sparkles:!
 
+## Installation
+
+You can install Jazzhands locally in the virtual environment of a Django project.
+
+```
+pip install -e git@github.com:caktus/jazzhands.git
```

Comment:
Shouldn't need ``-e`` unless someone wants to develop on jazzhands itself.

Reply:
That doesn't seem to be a valid argument to pip.  This works better:

    pip install 'git+https://github.com/caktus/jazzhands@python-improvements'